Benutzerdoku/Admin.tex

```diff
@@ -0,0 +1,85 @@
+\section{Adminsicht}
+
+\subsection{Installation}
+Doppelt klicken.
+
+
+
+\subsection{Start}
+	\includegraphics[width=\textwidth]{Bilder/Role-selection}
+	\\
+	\\
+Das ist das Startfenster der Anwendung. In diesem kann der Benutzer mit einem klick auf einen der angezeigten Buttons den Zugang auswählen welchen er nutzen möchte. Für die Benutzung der Anwendung als Administrator den Button \textit{Administrator} auswählen. 
```

Comment:
Bist du sicher, das es nicht auch mit figure geht? Dann bekommt jede Abbildung ne Nummer und man kann sagen in Abbildung `\ref{fig:theLabel}` bla bla. Beim Pflichtenheft hat sie darauf ganz großen Wert gelegt

Reply:
Sehe grad unten das `wrapfigure`. Kannst du diese Abbildung auch mit `wrapfigure umgeben` und ihr dann auch eine Caption und ein Label geben?

Reply:
1. ich könnte es machen aber dann fügt latex die Bilder an eine Falsche stelle 

2. eine caption und label wären möglich aber da die Bilder sowieso in dem Text sind wären sie nicht so dringend notwendig, außerdem da die anderen Bilder auch keine haben würde ich das in diesem fall auch nicht machen